src/components/button/Button.tsx

```diff
@@ -455,6 +460,13 @@ export function Button<T extends GroupTag = 'button'>({
           {rippleEnabled && <RippleGroup ref={rippleRef} />}
         </LinkButton>
       );
+    case 'plain':
+      return (
+        <PlainButton as={tag} {...buttonProps}>
+          {renderContent(spinnerColor)}
+          {rippleEnabled && <RippleGroup ref={rippleRef} />}
```

Comment:
Since this `rippleEnabled` will always be disabled on `plain` intent based on this
```
    ...
    rippleEnabled: !noRipple && intent !== 'plain',
```
I think the renderRipple logic is redundant here
```suggestion
        <PlainButton as={tag} {...buttonProps}>
          {renderContent(spinnerColor)}
```

Reply:
Agreed.